units/commit_queue.go

```diff
@@ -321,6 +320,28 @@ func makeMergePatch(ctx context.Context, pr *github.PullRequest, githubToken, pr
 		return nil, nil, errors.Wrap(err, "can't make patch")
 	}
 
+	// diff to be displayed in the UI
+	patchContent, summaries, err := thirdparty.GetGithubPullRequestDiff(ctx, githubToken, patchDoc.GithubPatchData)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "can't get diff")
+	}
+
+	patchFileID := fmt.Sprintf("%s_%s", patchDoc.Id.Hex(), patchDoc.Githash)
+	if err = db.WriteGridFile(patch.GridFSPrefix, patchFileID, strings.NewReader(patchContent)); err != nil {
+		return nil, nil, errors.Wrap(err, "failed to write patch file to db")
+	}
+
+	// empty string for name of the main patch
+	patchDoc.Patches = append(patchDoc.Patches, patch.ModulePatch{
+		ModuleName: "",
```

Comment:
nit: this is the default value, we don't need to specify it here.

Reply:
done.